.github/workflows/Cleanup Caches by a branch.yml

```diff
@@ -8,6 +8,10 @@ on:
   pull_request:
     branches: [main, 'ODS-*']
     types: [ closed ]
+
+concurrency:
```

Comment:
This workflow executes only when a PR is closed, so there will not be an in progress workflow to cancel. This change can be reverted.

Reply:
Revert it back . Thanks